cns/fakes/imdsclientfake.go

```diff
@@ -9,17 +9,21 @@ package fakes
 import (
 	"context"
 
+	"github.com/Azure/azure-container-networking/cns/imds"
 	"github.com/Azure/azure-container-networking/cns/wireserver"
 )
 
 const (
 	// HostPrimaryIP 10.0.0.4
 	HostPrimaryIP = "10.0.0.4"
 	// HostSubnet 10.0.0.0/24
-	HostSubnet = "10.0.0.0/24"
+	HostSubnet    = "10.0.0.0/24"
+	SimulateError = "simulate-error"
```

Comment:
I would probably just type this const as
```golang
SimulateError MockIMDSCtxKey = "simulate-error"
```
and don't do the type assertion later as usage

Reply:
I have fixed this.